enigma/src/main/java/cuchaz/enigma/analysis/StructureTreeOptions.java

```diff
@@ -0,0 +1,42 @@
+package cuchaz.enigma.analysis;
+
+import cuchaz.enigma.utils.I18n;
+
+public record StructureTreeOptions(
+        ObfuscationVisibility obfuscationVisibility,
+        DocumentationVisibility documentationVisibility,
+        SortingOrder sortingOrder) {
+
+    public enum SortingOrder {
+        DEFAULT,
+        A_Z,
+        Z_A;
+
+        @Override
+        public String toString() {
```

Comment:
I wouldn't use toString for getting any localized text (or in general, toString output reading data from anything outside of the object)... but I guess this is done because of the default behavior of Swing comboboxes to use toString for displaying the text, so, fine. Especially because I think the alternative is considerably more annoying to implement. *glares at Swing*

But what I would change however, is the dependency on the enum name instead of a separate string constant for the localization name, since it prevents you from being able to refactor the code (in this case, renaming the enum constants) without breaking anything.